README.md

```diff
@@ -30,3 +30,20 @@ To run lambda locally, use this script:
 sam local invoke --profile nypl-digital-dev -t sam.local.yml -e test/sample-events/b10001936.json
 ```
 
+## Running
+
+### Bulk Indexing
+
+The bulk-indexer can be used to index lots of records at once by direct BibService SQL query.
+
+For example, to reindex the first 10K NYPL bibs with marc 001 in QA:
+```
+node scripts/bulk-index.js --envfile config/qa-bulk-index.env --type bib --hasMarc 001 --nyplSource sierra-nypl --limit 10000
```

Comment:
👏🏻 incredible!! did we have this script anywhere before?

Reply:
This is added documentation, but the functionality has been hanging out in [this related PR](https://github.com/NYPL/research-catalog-indexer/pull/71) waiting to merge to main